### PR TITLE
Fix d_user.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
@gabe @brianhw

### CODE REVIEW

File: edx/analytics/tasks/location_per_course.py [hunk 1]
```diff
@@ -149,9 +149,15 @@ class ExternalLastCountryOfUserToHiveTask(ImportLastCountryOfUserToHiveTask):
...
+    def partition_spec(self):
+        return "{key}={value}".format(
+            key=self.partition.keys()[0],
+            value=self.partition.values()[0],
+        )
+
     def requires(self):
         yield ExternalURL(
-            url=url_path_join(self.warehouse_path, 'last_country_of_user', self.partition.path_spec) + '/'
+            url=url_path_join(self.warehouse_path, 'last_country_of_user', self.partition_spec()) + '/'
```
Comment (mulby) [inline edx/analytics/tasks/location_per_course.py:160]: Why can't we use `path_spec` here?

**Comment by mulby** [comment]
Just one question, mostly LGTM. :+1:

File: edx/analytics/tasks/location_per_course.py (hunk 1 above)
Comment (HassanJaveed84) [inline edx/analytics/tasks/location_per_course.py:160]: ImportIntoHiveTableTask defines partition as a simple dictionary:

``` @property
    def partition(self):
        """Provides name of Hive database table partition."""
        # The Luigi hive code expects partitions to be defined by dictionaries.
        return {'dt': self.partition_date}
```

Comment (mulby) [inline edx/analytics/tasks/location_per_course.py:160]: ah... tech debt. OK, let's ship it the way you have it then!

**Pull request merged**